Sources/PackageCollections/CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ target_link_libraries(PackageCollections PUBLIC
   TSCBasic
   TSCUtility
   Basics
+  Crypto
```

Comment:
@compnerd I try running the CMake build in local docker via `./Utilities/bootstrap build` with swift-crypto `main`, but for some reason `Crypto` is not found:

```
--- bootstrap: note: Building SwiftPM (with CMake)
[1/1] Linking Swift shared library lib/libPackageCollections.so
FAILED: lib/libPackageCollections.so Sources/PackageCollections/CMakeFiles/PackageCollections.dir/Model/Collection.swift.o Sources/PackageCollections/CMakeFiles/PackageCollections.dir/Model/CVE.swift.o Sources/PackageCollections/CMakeFiles/PackageCollections.dir/Model/License.swift.o Sources/PackageCollections/CMakeFiles/PackageCollections.dir/Model/Package.swift.o Sources/PackageCollections/CMakeFiles/PackageCollections.dir/Model/Search.swift.o Sources/PackageCollections/CMakeFiles/PackageCollections.dir/Model/TargetListResult.swift.o Sources/PackageCollections/CMakeFiles/PackageCollections.dir/Providers/GitHubPackageMetadataProvider.swift.o Sources/PackageCollections/CMakeFiles/PackageCollections.dir/Providers/JSONPackageCollectionProvider.swift.o Sources/PackageCollections/CMakeFiles/PackageCollections.dir/Providers/PackageCollectionProvider.swift.o Sources/PackageCollections/CMakeFiles/PackageCollections.dir/Providers/PackageMetadataProvider.swift.o Sources/PackageCollections/CMakeFiles/PackageCollections.dir/Storage/FilePackageCollectionsSourcesStorage.swift.o Sources/PackageCollections/CMakeFiles/PackageCollections.dir/Storage/PackageCollectionsSourcesStorage.swift.o Sources/PackageCollections/CMakeFiles/PackageCollections.dir/Storage/PackageCollectionsStorage.swift.o Sources/PackageCollections/CMakeFiles/PackageCollections.dir/Storage/SQLitePackageCollectionsStorage.swift.o Sources/PackageCollections/CMakeFiles/PackageCollections.dir/Storage/Trie.swift.o Sources/PackageCollections/CMakeFiles/PackageCollections.dir/API.swift.o Sources/PackageCollections/CMakeFiles/PackageCollections.dir/PackageCollections.swift.o Sources/PackageCollections/CMakeFiles/PackageCollections.dir/PackageCollections+Configuration.swift.o Sources/PackageCollections/CMakeFiles/PackageCollections.dir/PackageCollections+Storage.swift.o Sources/PackageCollections/CMakeFiles/PackageCollections.dir/PackageCollections+Validation.swift.o Sources/PackageCollections/CMakeFiles/PackageCollections.dir/Utility.swift.o swift/PackageCollections.swiftmodule 
: && /usr/bin/swiftc -output-file-map Sources/PackageCollections/CMakeFiles/PackageCollections.dir/output-file-map.json -incremental -j 6 -emit-library -o lib/libPackageCollections.so -module-name PackageCollections -module-link-name PackageCollections -emit-module -emit-module-path swift/PackageCollections.swiftmodule -emit-dependencies -DPackageCollections_EXPORTS -module-cache-path "/code/swift-package-manager/.build/x86_64-unknown-linux-gnu/bootstrap/module-cache" -g -swift-version 5 -I swift -I ../tsc/swift ../../../Sources/PackageCollections/Model/Collection.swift ../../../Sources/PackageCollections/Model/CVE.swift ../../../Sources/PackageCollections/Model/License.swift ../../../Sources/PackageCollections/Model/Package.swift ../../../Sources/PackageCollections/Model/Search.swift ../../../Sources/PackageCollections/Model/TargetListResult.swift ../../../Sources/PackageCollections/Providers/GitHubPackageMetadataProvider.swift ../../../Sources/PackageCollections/Providers/JSONPackageCollectionProvider.swift ../../../Sources/PackageCollections/Providers/PackageCollectionProvider.swift ../../../Sources/PackageCollections/Providers/PackageMetadataProvider.swift ../../../Sources/PackageCollections/Storage/FilePackageCollectionsSourcesStorage.swift ../../../Sources/PackageCollections/Storage/PackageCollectionsSourcesStorage.swift ../../../Sources/PackageCollections/Storage/PackageCollectionsStorage.swift ../../../Sources/PackageCollections/Storage/SQLitePackageCollectionsStorage.swift ../../../Sources/PackageCollections/Storage/Trie.swift ../../../Sources/PackageCollections/API.swift ../../../Sources/PackageCollections/PackageCollections.swift ../../../Sources/PackageCollections/PackageCollections+Configuration.swift ../../../Sources/PackageCollections/PackageCollections+Storage.swift ../../../Sources/PackageCollections/PackageCollections+Validation.swift ../../../Sources/PackageCollections/Utility.swift  -Xlinker -soname -Xlinker libPackageCollections.so  -L /code/swift-package-manager/.build/x86_64-unknown-linux-gnu/bootstrap/lib  -L /code/swift-package-manager/.build/x86_64-unknown-linux-gnu/bootstrap/lib  -L /code/swift-package-manager/.build/x86_64-unknown-linux-gnu/bootstrap/lib  -L /code/swift-package-manager/.build/x86_64-unknown-linux-gnu/tsc/lib  -L /code/swift-package-manager/.build/x86_64-unknown-linux-gnu/tsc/lib  -L /code/swift-package-manager/.build/x86_64-unknown-linux-gnu/tsc/lib -Xlinker -rpath -Xlinker /code/swift-package-manager/.build/x86_64-unknown-linux-gnu/bootstrap/lib:/code/swift-package-manager/.build/x86_64-unknown-linux-gnu/tsc/lib  -lCrypto  lib/libPackageModel.so  lib/libSourceControl.so  lib/libBasics.so  ../tsc/lib/libTSCUtility.so  ../tsc/lib/libTSCBasic.so  ../tsc/lib/libTSCLibc.so && :
/usr/bin/ld.gold: error: cannot find -lCrypto
clang-10: error: linker command failed with exit code 1 (use -v to see invocation)
<unknown>:0: error: link command failed with exit code 1 (use -v to see invocation)
ninja: build stopped: subcommand failed.
/usr/bin/ninja
--- bootstrap: error: Command '['/usr/bin/ninja']' returned non-zero exit status 1.
```

Any ideas on why?

The `find_package(Crypto CONFIG REQUIRED)` call in root `CMakeLists.txt` seems to have completed successfully. 

`libCrypto.so` exists at the proper location I think: 

```
> ls -al .build/x86_64-unknown-linux-gnu/swift-crypto/lib/
total 27008
drwxr-xr-x   5 ylee  staff      160 Jan 27 23:01 .
drwxr-xr-x  14 ylee  staff      448 Jan 27 23:01 ..
-rw-r--r--@  1 ylee  staff  9094506 Jan 27 23:01 libCCryptoBoringSSL.a
-rw-r--r--@  1 ylee  staff    23238 Jan 27 23:01 libCCryptoBoringSSLShims.a
-rwxr-xr-x   1 ylee  staff  4324720 Jan 27 23:01 libCrypto.so
```

At least I don't see swift-crypto's `CMakeLists.txt`s doing something different from `TSC`, `ArgumentParser`, etc.

Reply:
Found a potential cause: I'm not consistent with naming `SWIFT_CRYPTO_EXPORTS` vs. `CRYPTO_EXPORTS`. Will look into that.

Reply:
Indeed that was the issue.